### PR TITLE
[12.0][fix] base_tier_validation_formula: Add default

### PR DESCRIPTION
**Pull request by ps-tubtim**
Issue: https://github.com/OCA/server-ux/issues/137

cc: @kittiu

### CODE REVIEW

**Review by kittiu — APPROVED** [review]

**Comment by kittiu** [comment]
cc @LoisRForgeFlow , very small enh.

File: base_tier_validation_formula/models/tier_definition.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ class TierDefinition(models.Model):
         help="Write Python code that defines when this tier confirmation "
              "will be needed. The result of executing the expresion must be "
              "a boolean.",
-        default="""# Available locals:\n#  - rec: current record""",
+        default="""# Available locals:\n#  - rec: current record\nrec""",
```
Comment (etobella) [inline base_tier_validation_formula/models/tier_definition.py]: If it must be a boolean, why not True?

**Review by etobella — APPROVED** [review]
Just a comment, but not necessary
:+1:

File: base_tier_validation_formula/models/tier_definition.py (hunk 1 above)
Comment (ps-tubtim) [reply base_tier_validation_formula/models/tier_definition.py]: @etobella Thank you for suggestion. I changed this code.

**Comment by LoisRForgeFlow** [comment]
@ps-tubtim can you forward port?

**Comment by ps-tubtim** [comment]
@LoisRForgeFlow I forward port to https://github.com/OCA/server-ux/pull/143.

**Pull request merged**